client/src/components/helpers/Spacer.js

```diff
@@ -5,15 +5,18 @@ const styles = { padding: '15px 0', height: '1px' };
 
 const Comp = props => <div className='spacer' style={styles} {...props} />;
 
-const Spacer = ({ size = 1 }) =>
-  size === 1 ? (
-    <Comp />
-  ) : (
-    '#'
-      .repeat(size)
-      .split('')
-      .map((_, i) => <Comp key={`spacer_${i}`} />)
-  );
+const Spacer = ({ size = 1 }) => (
```

Comment:
This change can now be reverted, and this PR rebased with `next` to get similar once the first PR is merged.